tools/util/github.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"context"
+
+	"github.com/google/go-github/v32/github"
+	"golang.org/x/oauth2"
+)
+
+// GithubClientSource describes a function that can create a
+// github.Client without any other inputs.
+type GithubClientSource func() *github.Client
+
+func NewGithubClientSource(token string) GithubClientSource {
```

Comment:
I honestly no longer remember why I didn't just instantiate the client and pass it to the thing that needs it, instead of using a factory like this. Do you think we can eliminate the extra layer?

Reply:
It seems to work fine without it, I'll remove